packages/pwa/tests/lighthouserc.js

```diff
@@ -11,7 +11,12 @@ module.exports = {
         collect: {
             startServerCommand: 'npm run start',
             //TODO: Add key pages URLs here as they are implemented.
-            url: ['http://localhost:3000/'],
+            url: [
```

Comment:
You can probably remove that `TODO` above this line. You can potentially replace it with a `NOTE` telling people partners that they can edit this list to have lighthouse coverage on the pages that are important to them.